### DISCUSSION TITLE
Ignoring specific notifications

### DISCUSSION THREAD

**Discussion by ldelossa** [ErikReider/SwayNotificationCenter#276] [Q&A] [answered]
Hey, 

Now that I'm using the media player widget I don't really have a great purpose for handling spotify events (or other media player events.)

Is there a way to specifically ignore events, or at least, in the case of spotify events, make swaync treat these at automatically read.

**Comment by ErikReider** [comment, marked as answer]
Spotify allows the user to disable notifications in the settings :)

There's a section in the MAN 5 page about ignoring notifications. I don't remember what it's called right now and I'm on my phone 😅

**Reply by ldelossa** [reply]
thanks a lot!

**Reply by eeriemyxi** [reply]
> Spotify allows the user to disable notifications in the settings :)
> 
> There's a section in the MAN 5 page about ignoring notifications. I don't remember what it's called right now and I'm on my phone 😅

Hi, can you tell where to find the section on ignoring notifications? I can't seem to find it anywhere. That's because I got lucky with Spotify, however some apps don't always have similar configurable notifications.

One idea I had was to use the scripting functionality to match by app name then run `swaync --close-latest` but for some reason `swaync --close-latest` doesn't do anything? Only `swaync --close-all` seems to work but that's not really what I wanted.

# Update
I found it. It's called `notification-visibility`.

**Comment by treipatru** [comment]
I was looking for the same type of ignore functionality, specifically for bluetooth (bluez). Sure enough, there is a way to disable the notifications from being dispatched in the first place. Thanks for the suggestion, makes sense that you'd rather stop emitting the notification rather than ignoring it afterward.

However, looking through the man/5 page of swaync I can't find this ignore functionality described. If you ever have the time to drop a line here, it would be great.

Thank you for your work on swaync :pray:

**Reply by eeriemyxi** [reply]
`notification-visibility` in https://man.archlinux.org/man/swaync.5.en
